<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GpioWidget</class>
 <widget class="QWidget" name="GpioWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>58</width>
     <height>172</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="gpio_0">
        <property name="text">
         <string>gpio 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_0" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="gpio_1">
        <property name="text">
         <string>gpio 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_1" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="gpio_2">
        <property name="text">
         <string>gpio 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_2" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="gpio_3">
        <property name="text">
         <string>gpio 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_3" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="gpio_4">
        <property name="text">
         <string>gpio 4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_4" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="gpio_5">
        <property name="text">
         <string>gpio 5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_5" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="gpio_6">
        <property name="text">
         <string>gpio 6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_6" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="gpio_7">
        <property name="text">
         <string>gpio 7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_7" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>65</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>gpio 21</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_21" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>gpio 22</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_22" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>gpio 23</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_23" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>gpio 24</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_24" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>gpio 25</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_25" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>gpio 26</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_26" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>gpio 27</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_27" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>gpio 28</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_28" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>gpio 29</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="indicator_29" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
